lxd/main.go

```diff
@@ -15,6 +15,7 @@ import (
 
 // Global arguments
 var argAuto = gnuflag.Bool("auto", false, "")
+var argPreseed = gnuflag.Bool("preseed", false, "")
```

Comment:
Sounds reasonable to me. We could make "-" the default value (so you can still use "--preseed" without indicating anything else), hope gnuflag supports that.

If there's agreement, I'll change this, but in a follow-up branch.